src/main/kotlin/uk/gov/dluhc/printapi/service/PrintResponseProcessingService.kt

```diff
@@ -4,15 +4,18 @@ import uk.gov.dluhc.printapi.database.entity.Status.PENDING_ASSIGNMENT_TO_BATCH
 import uk.gov.dluhc.printapi.database.entity.Status.RECEIVED_BY_PRINT_PROVIDER
 import uk.gov.dluhc.printapi.database.entity.Status.SENT_TO_PRINT_PROVIDER
 import uk.gov.dluhc.printapi.database.repository.PrintDetailsRepository
+import uk.gov.dluhc.printapi.mapper.StatusMapper
+import uk.gov.dluhc.printapi.messaging.models.ProcessPrintResponseMessage
 import uk.gov.dluhc.printapi.printprovider.models.BatchResponse
 import uk.gov.dluhc.printapi.printprovider.models.BatchResponse.Status.SUCCESS
 import java.time.Clock
 import java.time.OffsetDateTime
 
-class PrintResponseBatchService(
+class PrintResponseProcessingService(
```

Comment:
Minor (and I can live with this if you or Fayi are doing it in a subsequent PR), but is it worth annotating this class with `@Service` and the mapper above with `@Component` ?

Reply:
Thanks, @NathanRussellValtech, for the suggestions. 

I intentionally didn't include `@Service` annotation here and the `@Component` annotation in the mapper, because the current tests for those classes are just unit tests and they are not asserting the annotation usage. 
Hopefully, we will have integration tests with the follow-up subtasks which will require those annotations to be present and we will add them accordingly at that point.

Reply:
That's cool; as long as you have it in hand 👍 😁